### ISSUE TITLE
Weird font issues on greek characters

### ISSUE THREAD

**Issue by George-Spanos** [zen-browser/desktop#3724]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I notice that on most websites that have Greek characters, there's a weird choice of fonts chosen by the browser. An example can be seen in the following URL.
https://www.eap.gr/to-eap/history/

If you open the above URL with Zen, you notice that the css font family is Helvetica, while the text is actually rendered as TeXGyreHeros-Regular. Regarding every character that has a hyphen (ώ,ί,ά etc), the font chosen is DejaVu Sans, which is the default Zen font, as I notice, which is correct. 
In order to check that this is zen specifically, I tested every Greek website with a vanilla installation of Firefox as well, and the text renders just fine, with the expected fonts. If you put them side by side, it's easy to notice the issue.

It might be an issue with the Helvetica font family, because I can browse other Greek sites (like [this one](https://www.news.gr/ellada/article/3785996/makelio-sti-glifada-taftopiithike-ke-o-defteros-nekros-ekkremouse-se-varos-tou-erithra-angelia-tis-interpol.html)) that use other fonts, and there are no weird fonts there.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.1

### What platform are you seeing the problem on?

Linux

**Comment by George-Spanos** [comment]
Yes, I can verify that it still exists. I have Zen version 1.7b (Firefox 134.0.1) (64-bit), on Ubuntu 24.04.

**Comment by George-Spanos** [comment]
I can verify that the issue still exists with 1.7.2b (Firefox 134.0.2) (64-bit).

**Comment by George-Spanos** [comment]
Hello. Yes, the issue still persists with Zen Browser 1.8.2b (Firefox 135.0.1), while not replicating on fireforx.
Here's [an example page](https://www.eap.gr/to-eap/history/) to compare.

**Comment by mr-cheffy** [comment]
## Closing this issue as part of a repo cleanup

Due to technical-debt we had in the past, we are currently clearing out old, duplicate, stale, and non-actionable issues to improve issue tracking and make it easier to focus on fixing real bugs.

If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

Thanks for your understanding and for helping us improve the project! 🚀

This issue can be reopened at: https://github.com/zen-browser/desktop/issues/new?template=bug_report.yml&title=Weird%20font%20issues%20on%20greek%20characters%0A
